test/converter.js

```diff
@@ -640,6 +640,7 @@ test('map switch schemas to unions', async function (test) {
 
   const unionType = {
     id: 'ParentOrChild',
+    type: 'object',
```

Comment:
After adding a validator for `"type": "object"` on the top-level schema, this test was failing.  Do schema with a `"switch"` key also require a `"type": "object"` key?  I assumed so, so I changed this dummy data, but documentation on `"switch"` is really hard to find!

Reply:
Adding `"type": "object"` seems reasonable to me.